governance.md

```diff
@@ -0,0 +1,137 @@
+
+Contributors may also review pull requests. This can be helpful, but their approval or disapproval is not decisive for merging or not merging PRs.
+
+As Contributors gain experience and familiarity with the project, their profile within, and commitment to, the community will increase. At some stage, they may find themselves being nominated for committership by an existing Peer or Owner.
+
+### Peers
+Peers are community members who have shown that they are committed to the continued development of the project through ongoing engagement with the community. Peers are given push/write access to the project’s GitHub repos.
+
+Peers:
+
+- Are expected to work on public branches of their forks and submit pull requests to the master branch.
+- Must submit pull requests for all their changes.
+- May label and close issues.
+- May merge compat data update pull requests only.
+- Have their non-data update work reviewed by [Owners](#Owners) before acceptance into the repository. Non-data pull requests are PRs that change the schema, update project meta-docs, the linter. or other infrastructure changes.
+- Should ask for super-review from other Peers or Owners on PRs that are disruptive or controversial.
+
+To become a Peer:
+
+- One must have shown a willingness and ability to participate in the project as a team player. Typically, a potential Peer will need to show that they have an understanding of and alignment with the project, its objectives, and its strategy.
+- Have contributed a significant amount of work to the project (e.g. in the form of PRs or PR reviews), thereby demonstrating their trustworthiness and commitment to the project.
+
+New Peers can be nominated by any existing Peers. Once they have been nominated, there will be a vote by the Owners.
+
+It is important to recognize that being a Peer is a privilege, not a right. That privilege must be earned and once earned it can be removed by the Owners. However, under normal circumstances committership exists for as long as the Peer wishes to continue engaging with the project. Inactive Peers might be marked as inactive or removed by the Owners and may re-enter when they choose to contribute again.
+
+#### List of peers
+- Rachel Andrew (@rachelandrew)
+- Daniel Beck (@ddbeck)
+- Ryan Johnson (@escattone), Mozilla
+- Jean-Yves Perrier (@teoli2003), Mozilla
+- Eric Shepherd (@a2sheppy), Mozilla
+- Irene Smith (@irenesmith), Mozilla
+- Estelle Weyl (@estelle), Mozilla
+- John Whitlock (@jwhitlock), Mozilla
+
+Some Peers are also [MDN Content Curators](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Documentation_topics_and_curators) and have thus earned the privilege to be a mdn-browser-compat-data peer, so that their expertise in a given content area (like CSS or HTML, etc.) can help improve the compat data for that same content area. Such peers are marked in the relevant folders using GitHub’s Code Owner mechanism.
```

Comment:
This paragraph and the following one could also be true of Owners, right? Is it worth saying "Peers and Owners" instead of just "Peers" in both cases, and moving them to a separate section after the "Peers" and "Owners" descriptions called something like "Additional paths to becoming a Peer or Owner"?

Reply:
I've moved this into its own section. Good suggestion, thank you! :)